### PR TITLE
Add numactl feature for mongo instances

### PR DESCRIPTION
**Pull request by F43RY**
**Description of the change**
With this new feature is possible to execute mongod/s instances prefixed with "numactl --interleave=all"

**Benefits**
As described in [MongoDB production Notes](https://docs.mongodb.com/manual/administration/production-notes/#configuring-numa-on-linux) this feature allow to enable numactl if desidered

**Possible drawbacks**

Possible performances issue if enabled and not supported or viceversa
**Applicable issues**

**Additional information**

### CODE REVIEW

File: 4.4/debian-10/prebuildfs/opt/bitnami/scripts/libos.sh [hunk 1]
```diff
@@ -251,9 +251,17 @@ convert_to_mb() {
 #########################
 debug_execute() {
     if ${BITNAMI_DEBUG:-false}; then
-        "$@"
+        if [ "${NUMACTL}" = true ]; then
+            numactl --interleave=all "$@"
```
Comment (javsalgar) [inline 4.4/debian-10/prebuildfs/opt/bitnami/scripts/libos.sh:255]: note that libos.sh is shared across all containers. Does it make sense to use numactl in the rest of the containers? It seems to me that it should go in libmongodb.sh instead, like in mongodb_execute or similar.

File: 4.4/debian-10/rootfs/opt/bitnami/scripts/mongodb-env.sh [hunk 2]
```diff
@@ -16,7 +16,10 @@ export BITNAMI_VOLUME_DIR="/bitnami"
 
 # Logging configuration
 export MODULE="${MODULE:-mongodb}"
-export BITNAMI_DEBUG="${BITNAMI_DEBUG:-false}"
+export BITNAMI_DEBUG="${BITNAMI_DEBUG:-true}"
+
+#numactl configuration
+export NUMACTL="${NUMACTL:-false}"
```
Comment (javsalgar) [inline 4.4/debian-10/rootfs/opt/bitnami/scripts/mongodb-env.sh:22]: Is there any use case where I wouldn't want to use numactl if possible?

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thank you so much! Changes required

File: 4.4/debian-10/prebuildfs/opt/bitnami/scripts/libos.sh (hunk 1 above)
Comment (F43RY) [reply 4.4/debian-10/prebuildfs/opt/bitnami/scripts/libos.sh:255]: OK, no problem.
I'm going to move modify to libmongodb.sh ASAP.

File: 4.4/debian-10/rootfs/opt/bitnami/scripts/mongodb-env.sh (hunk 2 above)
Comment (F43RY) [reply 4.4/debian-10/rootfs/opt/bitnami/scripts/mongodb-env.sh:22]: Hi,
according to [MongoDB Production notes](https://docs.mongodb.com/manual/administration/production-notes/#mongodb-and-numa-hardware):

> Running MongoDB on a system with Non-Uniform Memory Access (NUMA) can cause a number of operational problems, including slow performance for periods of time and high system process usage.

At the moment no automatism is checking if is possible to use numa on the machine so I think is better set a boolean that can (dis/en)able this feature accordingly.

Comment (javsalgar) [reply 4.4/debian-10/rootfs/opt/bitnami/scripts/mongodb-env.sh:22]: For consistency, I'd rather use `MONGODB_ENABLE_NUMACTL` as the variable name.

Comment (F43RY) [reply 4.4/debian-10/rootfs/opt/bitnami/scripts/mongodb-env.sh]: Done

File: 4.4/debian-10/Dockerfile [hunk 3]
```diff
@@ -25,8 +25,14 @@ ENV BITNAMI_APP_NAME="mongodb-sharded" \
     BITNAMI_IMAGE_VERSION="4.4.7-debian-10-r5" \
     PATH="/opt/bitnami/common/bin:/opt/bitnami/mongodb/bin:$PATH"
 
+#RUN mv /opt/bitnami/mongodb/bin/mongod /opt/bitnami/mongodb/bin/mongod_cmd && \
```
Comment (javsalgar) [inline 4.4/debian-10/Dockerfile:28]: Could you remove these comments?

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thanks! Small change

File: 4.4/debian-10/Dockerfile (hunk 3 above)
Comment (F43RY) [reply 4.4/debian-10/Dockerfile]: Done

File: 4.4/debian-10/Dockerfile [hunk 4]
```diff
@@ -30,3 +30,4 @@ EXPOSE 27017
 USER 1001
 ENTRYPOINT [ "/opt/bitnami/scripts/mongodb-sharded/entrypoint.sh" ]
 CMD [ "/opt/bitnami/scripts/mongodb-sharded/run.sh" ]
+#ENTRYPOINT [ "/usr/bin/tail", "-f", "/dev/null" ]
```
Comment (javsalgar) [inline 4.4/debian-10/Dockerfile:33]: Please remove this one as well

**Comment by javsalgar** [comment]
Thanks for the changes! I'll be in PTO until Tuesday. When I come back I'll proceed with the merging

**Pull request merged**